extensions/positron-python/python_files/posit/positron/positron_jedilsp.py

```diff
@@ -326,12 +326,22 @@ def start_tcp(self, host: str, port: int) -> None:
         self._stop_event = threading.Event()
         self._server = self.loop.run_until_complete(self.loop.create_server(self.lsp, host, port))
 
+        listeners = self._server.sockets
+        for socket in listeners:
+            addr, port = socket.getsockname()
+            if addr == host:
+                logger.info("LSP server is listening on port %d", port)
+                break
+        else:
+            port = 0
+            logger.warning("Could not determine server port")
```

Comment:
It's pretty fatal if you can't determine the port, not sure what that means for you all